sample_code/s3_to_s3.py

```diff
@@ -0,0 +1,53 @@
+import os
+import logging
+from parsons import Redshift, S3, utilities
+
+# Redshift setup - this assumes a Civis Platform parameter called "REDSHIFT"
+
+set_env_var(os.environ['REDSHIFT_PORT'])
+set_env_var(os.environ['REDSHIFT_DB'])
+set_env_var(os.environ['REDSHIFT_HOST'])
+set_env_var(os.environ['REDSHIFT_CREDENTIAL_USERNAME'])
+set_env_var(os.environ['REDSHIFT_CREDENTIAL_PASSWORD'])
+rs = Redshift()
+
+# AWS setup - this assumes a Civis Platform parameter called "AWS"
+
+set_env_var('S3_TEMP_BUCKET', 'parsons-tmc')
+set_env_var('AWS_ACCESS_KEY_ID', os.environ['AWS_USERNAME'])
+set_env_var('AWS_SECRET_ACCESS_KEY', os.environ['AWS_PASSWORD'])
+s3 = S3()
+
+# 2nd AWS setup - this assumes a Civis Platform parameter called "AWS_VENDOR"
+s3_vendor = S3(os.environ['AWS_VENDOR_ACCESS_KEY_ID'],
+            os.environ['AWS_VENDOR_SECRET_ACCESS_KEY'])
+
+# Logging
+
+logger = logging.getLogger(__name__)
+_handler = logging.StreamHandler()
+_formatter = logging.Formatter('%(levelname)s %(message)s')
+_handler.setFormatter(_formatter)
+logger.addHandler(_handler)
+logger.setLevel('INFO')
+
+# Let's write some code!
+
+# Get Vendor Bucket Information - this assumes it lives in Redshift
+bucket_guide = rs.query(f"select * from {table}")
```

Comment:
This seems like a pretty big "if", plus we don't have `table` defined anywhere before this. I'm wondering if it might not be better to use some more `S3` methods, like `list_buckets()` for this rather than assuming it'll be in Redshift?